base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
+  min-width: 414px;
+  height: 100vh;
+  background-color: RGB(25, 30, 40);
+  display: flex;
+  flex-direction: column;
+}
+
+.navigationBar {
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction: row` is the default for flex, so you don't need to set it.

Reply:
Good to know! Thanks

flex direction corrected (removed)